index.html

```diff
@@ -2021,6 +2021,33 @@ <h6 id="ActionStatus"><code>ActionStatus</code> object</h6>
                 <td>optional</td>
                 <td><code>string</code></td>
               </tr>
+              <tr class="rfc2119-table-assertion">
+                <td><code>timeRequested</code></td>
+                <td>
+                  A timestamp indicating the time at which the Thing received 
+                  the request to execute the action, in ISO format [[ISO8601-1]].
+                </td>
+                <td>optional</td>
+                <td><code>string</code></td>
+              </tr>
+              <tr class="rfc2119-table-assertion">
+                <td><code>timeCompleted</code></td>
+                <td>
+                  A timestamp indicating the time at which the Thing finished 
+                  executing the action, in ISO format [[ISO8601-1]].
+                </td>
+                <td>optional</td>
+                <td><code>string</code></td>
+              </tr>
+              <tr class="rfc2119-table-assertion">
+                <td><code>timeFailed</code></td>
+                <td>
+                  A timestamp indicating the time at which the Action failed to 
```

Comment:
Please consider to eliminate timeFailed. 
This information should be conveyed in the action response.